<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_register">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>440</y>
     <width>180</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Register/src/CHECKCONFIRM_W.png)}
QPushButton:hover{border-image:url(:/Register/src/CHECKCONFIRM_P.png) }
QPushButton:pressed{border-image: url(:/Register/src/CHECKCONFIRM_C.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Return/src/RETURN_W.png)}
QPushButton:hover{border-image: url(:/Return/src/RETURN_P.png)}
QPushButton:pressed{border-image: url(:/Return/src/RETURN_C.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_code">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_codeconfirm">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_account">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>290</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="index.qrc">:/Register/src/USERNAME_W_55_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_code">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="index.qrc">:/Register/src/CODE_W_45_25.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_checkname">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>45</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Register/src/CHECK_W.png)}
QPushButton:hover{border-image:url(:/Register/src/CHECK_P.png) }
QPushButton:pressed{border-image: url(:/Register/src/CHECK_C.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_useraccount">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="index.qrc">:/Register/src/CONFIRMCODE_W_60_25.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="index.qrc"/>
 </resources>
 <connections/>
</ui>
